notification_controller/controller/controller_test.go

```diff
@@ -119,6 +119,7 @@ func TestInit(t *testing.T) {
 		nil,
 		"my-secret",
 		"my-configmap",
+		false,
```

Comment:
I think we really oughta have some unit tests for when apps-in-any-namespace is enabled.